cli/azd/pkg/tools/docker/docker.go

```diff
@@ -82,6 +84,10 @@ func (d *docker) Build(
 		args = append(args, "-t", tagName)
 	}
 
+	if buildArgs != "" {
+		args = append(args, "--build-arg", buildArgs)
+	}
```

Comment:
Then here if buildArgs is defined we can go ahead and add an additional arg for each item